### PR TITLE
Document `/source/{project_name}/{package_name}/_meta` endpoints with OpenAPI

### PR DESCRIPTION
**Pull request by eduardoj**
### For reviewers

Access to the documented enpoints in the review-app through these links:
- [GET /source/{project_name}/{package_name}/_meta](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newsource_project_package_meta/apidocs-new/index.html#/Sources%20-%20Packages/get_source__project_name___package_name___meta)
- [PUT /source/{project_name}/{package_name}/_meta](https://obs-reviewlab.opensuse.org/eduardoj-apidocs_newsource_project_package_meta/apidocs-new/index.html#/Sources%20-%20Packages/put_source__project_name___package_name___meta)

### CODE REVIEW

File: src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      schema:
+        type: string
+      description: Set to `1` to force to retrieve the package metadata from the backend.
+      example: 1
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Return the package metadata for a specific revision number.
+      example:
```
Comment (saraycp) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml:20]: The `rev` example is empty here, therefore Swagger UI is displaying `Example : null`.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+          name: package_1
+          title: ''
+          description: Test package
+          devel:
+            project: home:Admin
+            package: package_2
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '400':
```
Comment (saraycp) [inline src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml:129]: No matter the 400 example I choose, Swagger UI is displaying

```
<?xml version="1.0" encoding="UTF-8"?>
<notagname/>
```

**Review by saraycp — CHANGES_REQUESTED** [review]

File: src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml (hunk 1 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml]: Ops, true. Fixed.

File: src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml (hunk 2 above)
Comment (eduardoj) [reply src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml:135]: Also fixed. Thanks!

**Review by saraycp — APPROVED** [review]

**Pull request merged**